html-css/week3/index3.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="./styles.css" />
+    <title>Preferences</title>
+</head>
+
+<body>
+
+    <form action="" method="GET">
+        <div class="navbar">
```

Comment:
`nav-bar` use kebab case with classes or if you want camel case then it should be `navBar`